frontend/src/libs/hooks/use-app-map/use-app-map.hook.ts

```diff
@@ -36,11 +43,32 @@ const useAppMap = ({
     const configMap = async (): Promise<void> => {
       await MapConnector.getInstance();
 
-      mapService.current = new MapConnector().getMapService({
-        mapElement: mapReference.current,
-        center: center ?? DEFAULT_CENTER,
-        zoom,
-      });
+      if (!points) {
+        mapService.current = new MapConnector().getMapService({
+          mapElement: mapReference.current,
+          center: center ?? DEFAULT_CENTER,
+          zoom,
+        });
+      } else if (points.length === 1) {
```

Comment:
We should show our intentions and try to answer a question: why exactly one point? Let's introduce a named helper for this.

Reply:
It's because if there is only one point in the list we can define this point as a center and init map with center and not get bounds.